fsspec/asyn.py

```diff
@@ -160,7 +160,7 @@ def get_loop():
     resource = None
     ResourceError = OSError
 else:
-    ResourceEror = resource.error
+    ResourceError = getattr(resource, "error", IOError)
```

Comment:
Note that starting from Python 3.3, `IOError` is an alias of `OSError`:
https://docs.python.org/3/library/exceptions.html#IOError

I have therefore changed `IOError` → `OSError` in #1279.